ads/google/a4a/utils.js

```diff
@@ -130,6 +136,9 @@ function buildAdUrl(
   const iframeDepth = iframeNestingDepth(global);
   const dtdParam = {name: 'dtd'};
   const adElement = a4a.element;
+  if (ValidAdContainerTypes.indexOf(adElement.parentElement.tagName) >= 0) {
+    queryParams.push({name: 'amp_ct', value: adElement.parentElement.tagName});
+  }
   const allQueryParams = queryParams.concat(
```

Comment:
I think this would be much easier to test once my [PR related to missing AdSense parameters](https://github.com/ampproject/amphtml/pull/6531) gets merged. In that PR, I've already written a couple of tests that verify the URL parameters, and it would be relatively trivial to add one to test for amp_ct.